### PR TITLE
svelte: Update Tree-sitter grammar

### PR DESCRIPTION
**Pull request by matubu**
Before:
Screenshot from 2024-09-03 20-36-48

After:
Screenshot from 2024-09-03 20-43-44

Release Notes:

- N/A

### CODE REVIEW

File: .zed/settings.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "hard_tabs": false,
```
Comment (maxdeviant) [inline .zed/settings.json:2]:
```suggestion
```

**Review by maxdeviant — APPROVED** [review]
Thanks!

**Comment by AlbertMarashi** [comment]
This fixed https://github.com/zed-industries/zed/issues/17310
but broke some other stuff!

image

**Comment by AlbertMarashi** [comment]
https://github.com/zed-industries/zed/pull/17404

**Pull request merged**